lotus/settings.py

```diff
@@ -13,6 +13,7 @@ class Settings:
 
     # Cache settings
     enable_cache: bool = False
```

Comment:
Can `enable_cache` be renamed to `enable_messsage_cache` so its clear what the distinction is

Reply:
yea I can do that